lib/desired-caps.js

```diff
@@ -380,6 +380,10 @@ const desiredCapConstraints = /** @type {const} */ ({
   appTimeZone: {
     isString: true,
   },
+  pageLoadStrategy: {
```

Comment:
I assume it should be already defined in the base driver

Reply:
Yea, I thought so, but it was used to check appium prefix stuff only:

With current master:
```
[XCUITestDriver@b618] The following provided capabilities were not recognized by this driver:
[XCUITestDriver@b618]   pageLoadStrategy
[XCUITestDriver@b618] The desired capabilities include neither an app nor a bundleId. WebDriverAgent will be started without the default app
[XCUITestDriver@b618] Session created with session id: 609c241a-fcf6-44a6-be40-c5eff0cde7fe
```

Reply:
weird...
I would expect all standard W3C caps to be defined there. I can also observe corresponding typedefs present:
https://github.com/appium/appium/blob/master/packages/types/lib/standard-caps.ts

Reply:
it looks like there is a bug in typedefs. Not sure where exactly yet...

Reply:
Created for future investigation https://github.com/appium/appium/issues/20273